src/loki/route_action.cc

```diff
@@ -102,12 +106,10 @@ void loki_worker_t::route(Api& request) {
     for (size_t i = 0; i < locations.size(); ++i) {
       const auto& correlated = projections.at(locations[i]);
       PathLocation::toPBF(correlated, options.mutable_locations(i), *reader);
-      // TODO: get transit level for transit costing
-      // TODO: if transit send a non zero radius
       if (!connectivity_map) {
         continue;
       }
-      auto colors = connectivity_map->get_colors(TileHierarchy::levels().back().level, correlated, 0);
+      auto colors = connectivity_map->get_colors(connectivity_level, correlated, connectivity_radius);
```

Comment:
pass the full `TileLevel`, we need access to level 2 or 3 tiles and need to know which level it was